pkg/webui/components/link/index.js

```diff
@@ -80,9 +80,9 @@ const Link = React.forwardRef((props, ref) => {
 
   if (disabled) {
     return (
-      <span className={classnames(classNames, style.disabled)} role={role}>
+      <button className={classnames(classNames, style.disabled)} disabled={disabled} role={role}>
```

Comment:
What's the reason to change this globally? Links are not buttons.

Reply:
When using `<Button.Link />` component, disabled styles are not applied because in the button styles we are using `:disabled` and in the link it is `.disabled` so I made it much more consistent. From testing this I made sure that this works with regular links too. Also there were some issue when hovering disabled links before.

https://github.com/user-attachments/assets/056dbec0-cf29-417a-9096-ec515771a3b7